README.md

```diff
@@ -66,16 +84,20 @@ Next follow these instructions.
 
 Follow the given instructions for Login into the app.
 
+
+1. To create the superuser run:
+=======
 1. You can register a new user by setting up SendGrid details in the `settings.py`, you can take a reference from [here](https://sendgrid.com/docs/for-developers/sending-email/integrating-with-the-smtp-api/)
 
 2. To create the superuser run:
+
    ```
    python manage.py createsuperuser
    ````
    Fill up the things it asked to and then Login into the app.
 
 ## Contributing
-Please read the Contibuting guidelines, [Code of Conduct](https://github.com/anitab-org/open-source-programs-backend/blob/develop/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/open-source-programs-backend/blob/develop/REPORTING_GUIDELINES.md)
+Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Conduct](https://github.com/anitab-org/open-source-programs-backend/blob/develop/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/open-source-programs-backend/blob/develop/REPORTING_GUIDELINES.md)
```

Comment:
@khushishikhu Can you please make paths to reporting guidelines & code of conduct also relative similar to contributing guidelines.

Reply:
I don't think that it would be same as it is in the contributing guidelines

> @khushishikhu Can you please make paths to reporting guidelines & code of conduct also relative similar to contributing guidelines.